site/docs/id/README.md

```diff
@@ -86,5 +85,13 @@ Berhasil! :tada:
 
 ---
 
-grammY mendukung API Bot Telegram versi 6.6 yang [dirilis](https://core.telegram.org/bots/api#march-9-2023) pada tanggal 9 Maret 2023.
+<ClientOnly>
+  <ThankYou :s="['Thank you ', 'Someone', ' for being a contributor to grammY.']" />
```

Comment:
@fadzikri @CikiMomogi this the only line that needs to be translated to Indonesian